lib/types/index.d.ts

```diff
@@ -621,12 +621,12 @@ export namespace Rule {
 			LangOptions: Linter.LanguageOptions;
 			Code: SourceCode;
 			RuleOptions: any[];
-			Visitor: NodeListener;
+			Visitor: RuleListener;
```

Comment:
`RuleModule` is the type of a rule in eslintrc, where only JavaScript is supported. I think it's correct to use `NodeListener` in this case, because it's JavaScript-specific.